doc/gov/chapters/chapter02.md

```diff
@@ -13,26 +13,36 @@
 <a name="2.1"></a>
 ## 2.1 Introduction
 
-As an open-source community organization, CNTT has developed the following process to ensure delivery on CNTT objectives. This process has several goals. First, inform new participants with the background and overview of this open-source initiative. Second, communicate the organizational and legal structures upon which the community operates. And third, help new participants become fully engage & contributing effectively to the goals of CNTT via its various work streams.
+As an open-source community organization, CNTT has developed the following process to ensure delivery on the CNTT objectives. This process has several goals. First, to inform new participants with the background and overview of this open-source initiative. Second, to communicate the organizational and legal structures under which the community operates. And third, to help new participants to quickly become fully engaged in the CNTT work and to be contributing effectively to the work streams they are interested in.
 
 
 
 <a name="2.2"></a>
 ## 2.2 Request Onboarding Info Packet
 
-[Welcome Message Here]
+[Sample Welcome Message Here] 
+Welcome to the CNTT group –
+
+It is wonderful that [Company and person] has decided to join in the effort to make building the infrastructure to support Telecom workloads more robust and standardized. We believe that these efforts will benefit not only the operators, but also the vendors, and ultimately all our customers as well, by hopefully making it easier and simpler to incorporate new technology into our systems.
```

Comment:
should we make this personal as in "It is wonderful that you have decided to join ..." -- the salutation should have the name of the person and organisation.

Also, should the first sentence reference "common infrastructure reference platform"

Can we modify the second sentence, "We believe that these efforts will benefit the operators and vendors, and ultimately our customers, by making it easier and simpler to incorporate new technology into our systems."

Reply:
I believe those has been proposed as commit proposals.